frontend/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:21.6.2
+
```

Comment:
Add the line `ENV CHOKIDAR_USEPOLLING=true` to the Dockerfile to ensure that changes in the code are automatically reflected in the container when using Docker on Windows.